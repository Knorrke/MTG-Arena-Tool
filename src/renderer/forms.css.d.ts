/* eslint-disable */
// GENERATED FILE; DO NOT EDIT
declare namespace FormsCssModule {
  export interface IFormsCss {
    "form-authenticate": string;
    "form-button": string;
    "form-container": string;
    "form-error": string;
    "form-icon": string;
    "form-input-container": string;
    "form-label": string;
    "form-options": string;
    formAuthenticate: string;
    formButton: string;
    formContainer: string;
    formError: string;
    formIcon: string;
    formInputContainer: string;
    formLabel: string;
    formOptions: string;
  }
}

declare const FormsCssModule: FormsCssModule.IFormsCss & {
  /** WARNING: Only available when `css-loader` is used without `style-loader` or `mini-css-extract-plugin` */
  locals: FormsCssModule.IFormsCss;
};

export = FormsCssModule;
