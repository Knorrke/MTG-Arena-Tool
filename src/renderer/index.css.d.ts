declare namespace IndexCssModule {
  export interface IIndexCss {
    about: string;
    active: string;
    archiveLabel: string;
    archive_label: string;
    arrowDown: string;
    arrowUp: string;
    arrow_down: string;
    arrow_up: string;
    authLabel: string;
    auth_label: string;
    authenticate: string;
    boDeckstabCost: string;
    boExploreCost: string;
    bo_deckstab_cost: string;
    bo_explore_cost: string;
    butContainerLabel: string;
    butDrop: string;
    butJoin: string;
    but_container_label: string;
    but_drop: string;
    but_join: string;
    buttonEdit: string;
    buttonLong: string;
    buttonReset: string;
    buttonSimple: string;
    buttonSimpleDisabled: string;
    buttonThin: string;
    button_edit: string;
    button_long: string;
    button_reset: string;
    button_simple: string;
    button_simple_disabled: string;
    button_thin: string;
    buttonsContainer: string;
    buttons_container: string;
    "card-wr-item": string;
    "card-wr-line": string;
    "card-wr-line-avg-first": string;
    "card-wr-line-avg-turn": string;
    "card-wr-line-card": string;
    "card-wr-line-mulls": string;
    "card-wr-line-sided-hand-wr": string;
    "card-wr-line-sided-in": string;
    "card-wr-line-sided-in-wr": string;
    "card-wr-line-sided-out": string;
    "card-wr-line-sided-out-wr": string;
    "card-wr-line-wr": string;
    "card-wr-stats": string;
    cardListsList: string;
    cardWrItem: string;
    cardWrLine: string;
    cardWrLineAvgFirst: string;
    cardWrLineAvgTurn: string;
    cardWrLineCard: string;
    cardWrLineMulls: string;
    cardWrLineSidedHandWr: string;
    cardWrLineSidedIn: string;
    cardWrLineSidedInWr: string;
    cardWrLineSidedOut: string;
    cardWrLineSidedOutWr: string;
    cardWrLineWr: string;
    cardWrStats: string;
    card_lists_list: string;
    cardlist: string;
    centered: string;
    centeredSettingContainer: string;
    centered_setting_container: string;
    "change-add": string;
    "change-remove": string;
    "change-warning": string;
    changeAdd: string;
    changeRemove: string;
    changeWarning: string;
    chartContainer: string;
    chart_container: string;
    checkContainer: string;
    checkContainerBut: string;
    checkContainerCentered: string;
    check_container: string;
    check_container_but: string;
    check_container_centered: string;
    checkmark: string;
    common: string;
    completionTable: string;
    completionTableCard: string;
    completionTableColorTitle: string;
    completionTableRarityTitle: string;
    completion_table: string;
    completion_table_card: string;
    completion_table_color_title: string;
    completion_table_rarity_title: string;
    constructedRank: string;
    constructed_rank: string;
    copyButton: string;
    copy_button: string;
    dateCont: string;
    "deck-change": string;
    "deck-changes-container": string;
    "deck-changes-expand": string;
    deckChange: string;
    deckChangesContainer: string;
    deckChangesExpand: string;
    deckFiltersCont: string;
    deckName: string;
    deckTag: string;
    deckTagClose: string;
    deckTagInput: string;
    deckTagsContainer: string;
    deckTile: string;
    deckTopColors: string;
    deckVisualCard: string;
    deckVisualCardImg: string;
    deckVisualCardSide: string;
    deckVisualStats: string;
    deckVisualTile: string;
    deckVisualTileSide: string;
    deck_name: string;
    deck_tag: string;
    deck_tag_close: string;
    deck_tag_input: string;
    deck_tags_container: string;
    deck_tile: string;
    deck_top_colors: string;
    deck_visual_card: string;
    deck_visual_card_img: string;
    deck_visual_card_side: string;
    deck_visual_stats: string;
    deck_visual_tile: string;
    deck_visual_tile_side: string;
    decklist: string;
    "decklist-colors": string;
    "decklist-name": string;
    decklistColors: string;
    decklistDivided: string;
    decklistName: string;
    decklistTop: string;
    decklist_divided: string;
    decklist_top: string;
    "decks-table-deck-archive": string;
    "decks-table-deck-inner": string;
    "decks-table-deck-item": string;
    "decks-table-deck-tile": string;
    "decks-table-deck-unarchive": string;
    "decks-table-wrapper": string;
    decksTableDeckArchive: string;
    decksTableDeckInner: string;
    decksTableDeckItem: string;
    decksTableDeckTile: string;
    decksTableDeckUnarchive: string;
    decksTableWrapper: string;
    decksTop: string;
    decks_top: string;
    deleteItem: string;
    delete_item: string;
    dialog: string;
    dialogContent: string;
    dialogWrapper: string;
    dialog_content: string;
    dialog_wrapper: string;
    disabled: string;
    discordBut: string;
    discordIcon: string;
    discordLink: string;
    discordMessage: string;
    discord_but: string;
    discord_icon: string;
    discord_link: string;
    discord_message: string;
    donateLink: string;
    donate_link: string;
    "draft-card": string;
    "draft-card-cont": string;
    "draft-card-picked": string;
    "draft-card-rank": string;
    "draft-container": string;
    "draft-deck-view": string;
    "draft-title": string;
    "draft-view": string;
    draftCard: string;
    draftCardCont: string;
    draftCardPicked: string;
    draftCardRank: string;
    draftContainer: string;
    draftDeckView: string;
    draftNavContainer: string;
    draftNavNext: string;
    draftNavPrev: string;
    draftTitle: string;
    draftView: string;
    draft_nav_container: string;
    draft_nav_next: string;
    draft_nav_prev: string;
    draft_title: string;
    dragger: string;
    economy: string;
    economyCard: string;
    economyDateSelect: string;
    economyDayStats: string;
    economyDelta: string;
    economyDown: string;
    economyExp: string;
    economyGems: string;
    economyGold: string;
    economyIconMargin: string;
    economyMasteryMed: string;
    economyMetric: string;
    economyOrb: string;
    economyRowHidden: string;
    economySkinArt: string;
    economySub: string;
    economyTicket: string;
    economyTicketMed: string;
    economyTitle: string;
    economyUp: string;
    economyVanity: string;
    economyVault: string;
    economyWc: string;
    economyWcMed: string;
    economy_card: string;
    economy_day_stats: string;
    economy_delta: string;
    economy_down: string;
    economy_exp: string;
    economy_gems: string;
    economy_gold: string;
    economy_mastery_med: string;
    economy_metric: string;
    economy_orb: string;
    economy_row_hidden: string;
    economy_skin_art: string;
    economy_sub: string;
    economy_ticket: string;
    economy_ticket_med: string;
    economy_title: string;
    economy_up: string;
    economy_vanity: string;
    economy_vault: string;
    economy_wc: string;
    economy_wc_med: string;
    "expand-arrow": string;
    expandArrow: string;
    exploreButtonsContainer: string;
    exploreButtonsRow: string;
    exploreButtonsTop: string;
    exploreWcInput: string;
    explore_buttons_container: string;
    explore_buttons_row: string;
    explore_buttons_top: string;
    explore_wc_input: string;
    exportDeck: string;
    exportDeckStandard: string;
    filterColumn: string;
    filterPanelRoot: string;
    filterPanelSelectMargin: string;
    filter_column: string;
    filter_panel_root: string;
    filter_panel_select_margin: string;
    flexEnd: string;
    flexLeft: string;
    flexTop: string;
    flex_end: string;
    flex_top: string;
    gameNext: string;
    gamePrev: string;
    gameSwap: string;
    game_next: string;
    game_prev: string;
    game_swap: string;
    gamestatsSide: string;
    gamestatsSubtitle: string;
    gamestats_side: string;
    gamestats_subtitle: string;
    gitLink: string;
    git_link: string;
    hover: string;
    hoverLabel: string;
    hover_label: string;
    icon0: string;
    icon1: string;
    icon2: string;
    icon3: string;
    icon4: string;
    icon5: string;
    icon7: string;
    icon8: string;
    iconSearchBooster: string;
    iconSearchInclusive: string;
    iconSearchIncomplete: string;
    iconSearchMulti: string;
    iconSearchNew: string;
    iconSearchUnowned: string;
    "icon_-1": string;
    icon_0: string;
    icon_1: string;
    icon_2: string;
    icon_3: string;
    icon_4: string;
    icon_5: string;
    icon_7: string;
    icon_8: string;
    icon_search_booster: string;
    icon_search_inclusive: string;
    icon_search_incomplete: string;
    icon_search_multi: string;
    icon_search_new: string;
    icon_search_unowned: string;
    innerDiv: string;
    inner_div: string;
    inputContainer: string;
    inputContainerExplore: string;
    inputLock: string;
    inputLoginContainer: string;
    input_container: string;
    input_container_explore: string;
    input_lock: string;
    input_login_container: string;
    inventoryCard: string;
    inventoryCardAetherized: string;
    inventoryCardImg: string;
    inventoryCardInfinityBlue: string;
    inventoryCardInfinityGray: string;
    inventoryCardInfinityGreen: string;
    inventoryCardInfinityOrange: string;
    inventoryCardQuantity: string;
    inventoryCardQuantityBlue: string;
    inventoryCardQuantityContainer: string;
    inventoryCardQuantityGray: string;
    inventoryCardQuantityGreen: string;
    inventoryCardQuantityOrange: string;
    inventoryCardSettings: string;
    inventoryCardSettingsImg: string;
    inventory_card: string;
    inventory_card_aetherized: string;
    inventory_card_img: string;
    inventory_card_infinity_blue: string;
    inventory_card_infinity_gray: string;
    inventory_card_infinity_green: string;
    inventory_card_infinity_orange: string;
    inventory_card_quantity: string;
    inventory_card_quantity_blue: string;
    inventory_card_quantity_container: string;
    inventory_card_quantity_gray: string;
    inventory_card_quantity_green: string;
    inventory_card_quantity_orange: string;
    inventory_card_settings: string;
    inventory_card_settings_img: string;
    it10: string;
    it11: string;
    itemSelected: string;
    item_selected: string;
    joinDecklist: string;
    join_decklist: string;
    keycombDesc: string;
    keycomb_desc: string;
    lastSeenClock: string;
    last_seen_clock: string;
    libraryCard: string;
    libraryFootnote: string;
    libraryHand: string;
    libraryHeader: string;
    libraryList: string;
    libraryStat: string;
    library_card: string;
    library_footnote: string;
    library_hand: string;
    library_header: string;
    library_list: string;
    library_stat: string;
    light: string;
    limitedRank: string;
    limited_rank: string;
    lineBottomBorder: string;
    lineDark: string;
    lineLight: string;
    lineLighter: string;
    line_bottom_border: string;
    line_dark: string;
    line_light: string;
    line_lighter: string;
    link: string;
    listDeck: string;
    listDeckLast: string;
    listDeckName: string;
    listDeckNameIt: string;
    listDeckRecord: string;
    listDeckRightIt: string;
    listDeckWinrate: string;
    listDraftShare: string;
    listEconomy: string;
    listEconomyAwarded: string;
    listEconomyTitle: string;
    listEconomyTop: string;
    listEventExpand: string;
    listEventPhase: string;
    listEventPhaseRed: string;
    listFill: string;
    listItemArchive: string;
    listItemCenter: string;
    listItemContainer: string;
    listItemImage: string;
    listItemLeft: string;
    listItemRight: string;
    listItemUnarchive: string;
    listLogCantShare: string;
    listLogShare: string;
    listMatch: string;
    listMatchReplay: string;
    listMatchResult: string;
    listMatchResultScore: string;
    listMatchTime: string;
    listMatchTimeTop: string;
    listMatchTitle: string;
    list_deck: string;
    list_deck_last: string;
    list_deck_name: string;
    list_deck_name_it: string;
    list_deck_record: string;
    list_deck_right_it: string;
    list_deck_winrate: string;
    list_draft_share: string;
    list_economy: string;
    list_economy_awarded: string;
    list_economy_title: string;
    list_economy_top: string;
    list_event_expand: string;
    list_event_phase: string;
    list_event_phase_red: string;
    list_fill: string;
    list_item_archive: string;
    list_item_center: string;
    list_item_container: string;
    list_item_image: string;
    list_item_left: string;
    list_item_right: string;
    list_item_unarchive: string;
    list_log_cant_share: string;
    list_log_share: string;
    list_match: string;
    list_match_replay: string;
    list_match_result: string;
    list_match_result_score: string;
    list_match_time: string;
    list_match_time_top: string;
    list_match_title: string;
    mainStats: string;
    main_stats: string;
    manaCurve: string;
    manaCurveColumn: string;
    manaCurveColumnNumber: string;
    manaCurveCosts: string;
    manaCurveNumber: string;
    manaCurveNumbers: string;
    manaCurveTag: string;
    manaFilter: string;
    manaFilterOn: string;
    manaFilterSearch: string;
    manaFiltersExplore: string;
    mana_curve: string;
    mana_curve_column: string;
    mana_curve_column_number: string;
    mana_curve_costs: string;
    mana_curve_number: string;
    mana_curve_numbers: string;
    mana_curve_tag: string;
    mana_filter: string;
    mana_filter_on: string;
    mana_filter_search: string;
    mana_filters_explore: string;
    marginLeft: string;
    matchPlayerName: string;
    matchPlayerWin: string;
    match_player_name: string;
    match_player_win: string;
    matchesTags: string;
    matchesTop: string;
    matches_tags: string;
    matches_top: string;
    messageBig: string;
    messageCenter: string;
    messageCenterOffline: string;
    messageSub: string;
    messageSub15: string;
    messageSub16: string;
    messageUnlink: string;
    messageUpdates: string;
    message_big: string;
    message_center: string;
    message_center_offline: string;
    message_sub: string;
    message_sub_15: string;
    message_sub_16: string;
    message_unlink: string;
    message_updates: string;
    movingUx: string;
    moving_ux: string;
    mythic: string;
    n0: string;
    n1: string;
    n2: string;
    n3: string;
    n4: string;
    navSelected: string;
    nav_selected: string;
    notification: string;
    notplayed: string;
    notplayedtext: string;
    onthedraw: string;
    onthedrawtext: string;
    ontheplay: string;
    ontheplaytext: string;
    openButton: string;
    openDeck: string;
    openMatches: string;
    open_button: string;
    overflowUx: string;
    overflowUxMain: string;
    overflow_ux: string;
    overflow_ux_main: string;
    overlayCardQuantity: string;
    overlayCurrent: string;
    overlayIcon: string;
    overlayLabel: string;
    overlayNext: string;
    overlayPrev: string;
    overlaySection: string;
    overlaySectionSelectorCont: string;
    overlay_card_quantity: string;
    overlay_current: string;
    overlay_icon: string;
    overlay_label: string;
    overlay_next: string;
    overlay_prev: string;
    overlay_section: string;
    overlay_section_selector_cont: string;
    pagingActive: string;
    pagingButton: string;
    pagingButtonDisabled: string;
    pagingContainer: string;
    pagingText: string;
    paging_active: string;
    paging_button: string;
    paging_button_disabled: string;
    paging_container: string;
    paging_text: string;
    patreonLink: string;
    patreonLinkThin: string;
    patreon_link: string;
    patreon_link_thin: string;
    pie: string;
    pieContainer: string;
    pieContainerOuter: string;
    pie_container: string;
    pie_container_outer: string;
    rank: string;
    "rank-bullet-dark": string;
    "rank-bullet-light": string;
    "rank-bullets": string;
    "rank-to-right": string;
    rankBulletDark: string;
    rankBulletLight: string;
    rankBullets: string;
    rankFilter: string;
    rankFilterOn: string;
    rankToRight: string;
    rank_filter: string;
    rank_filter_on: string;
    ranks16: string;
    ranksHistory: string;
    ranksHistoryBadge: string;
    ranksHistoryTitle: string;
    ranks_16: string;
    ranks_history: string;
    ranks_history_badge: string;
    ranks_history_title: string;
    rare: string;
    "rarity-overlay": string;
    rarityFilter: string;
    rarityFilterOn: string;
    rarityOverlay: string;
    rarity_filter: string;
    rarity_filter_on: string;
    "react-autosuggest__container": string;
    "react-autosuggest__input": string;
    "react-autosuggest__suggestion--highlighted": string;
    "react-autosuggest__suggestions-list": string;
    reactAutosuggestContainer: string;
    reactAutosuggestInput: string;
    reactAutosuggestSuggestionHighlighted: string;
    reactAutosuggestSuggestionsList: string;
    reactTableBody: string;
    reactTableBodyNoAdjust: string;
    reactTableBodyRow: string;
    reactTableHead: string;
    reactTableHeadContainer: string;
    reactTableSearchCont: string;
    reactTableToggles: string;
    reactTableWrap: string;
    react_table_body: string;
    react_table_body_no_adjust: string;
    react_table_body_row: string;
    react_table_head: string;
    react_table_head_container: string;
    react_table_search_cont: string;
    react_table_toggles: string;
    react_table_wrap: string;
    releaseNotesLink: string;
    release_notes_link: string;
    roundCard: string;
    round_card: string;
    select: string;
    "select-hidden": string;
    "select-options": string;
    "select-styled": string;
    selectButton: string;
    selectContainer: string;
    selectHidden: string;
    selectOption: string;
    selectOptions: string;
    selectOptionsContainer: string;
    selectStyled: string;
    selectTitle: string;
    select_button: string;
    select_container: string;
    select_option: string;
    select_options_container: string;
    select_title: string;
    setFilter: string;
    setFilterOn: string;
    setLogo: string;
    setLogoMed: string;
    setStats: string;
    set_filter: string;
    set_filter_on: string;
    set_logo: string;
    set_logo_med: string;
    set_stats: string;
    setsContainer: string;
    sets_container: string;
    "settings-select": string;
    settingsNav: string;
    settingsNote: string;
    settingsPage: string;
    settingsRow: string;
    settingsSection: string;
    settingsSelect: string;
    settingsTitle: string;
    settings_nav: string;
    settings_note: string;
    settings_page: string;
    settings_row: string;
    settings_section: string;
    settings_title: string;
    shareInputContainer: string;
    shareSubtitle: string;
    shareTitle: string;
    share_input_container: string;
    share_subtitle: string;
    share_title: string;
    shortcutsGrid: string;
    shortcutsLine: string;
    shortcuts_grid: string;
    shortcuts_line: string;
    "sidebar-dragger": string;
    "sidebar-main": string;
    sidebarColumnL: string;
    sidebarColumnR: string;
    sidebarDragger: string;
    sidebarMain: string;
    sidebarSeparator: string;
    sidebar_column_l: string;
    sidebar_column_r: string;
    sidebar_separator: string;
    sideboarderContainer: string;
    sideboarder_container: string;
    slidecontainer: string;
    slider: string;
    sliderMarkHor: string;
    sliderMarkOuter: string;
    sliderMarkText: string;
    sliderMarksContainerHor: string;
    slider_mark_hor: string;
    slider_mark_outer: string;
    slider_mark_text: string;
    slider_marks_container_hor: string;
    small: string;
    sortAsc: string;
    sortDesc: string;
    sortDiv: string;
    sortLabel: string;
    sortSelect: string;
    sort_asc: string;
    sort_desc: string;
    spacer180: string;
    stats: string;
    statsRarityIcon: string;
    statsSetBar: string;
    statsSetCompletion: string;
    statsSetCompletionRow: string;
    statsSetDetails: string;
    statsSetIcon: string;
    statsSidebar: string;
    stats_rarity_icon: string;
    stats_set_bar: string;
    stats_set_completion: string;
    stats_set_completion_row: string;
    stats_set_details: string;
    stats_set_icon: string;
    stats_sidebar: string;
    statusBlack: string;
    statusGreen: string;
    statusItem: string;
    statusLocked: string;
    statusRed: string;
    statusYellow: string;
    status_black: string;
    status_green: string;
    status_item: string;
    status_locked: string;
    status_red: string;
    status_yellow: string;
    swapIcon: string;
    swap_icon: string;
    switch: string;
    "switch-container": string;
    "switch-label": string;
    switchContainer: string;
    switchLabel: string;
    switchslider: string;
    textCentered: string;
    text_centered: string;
    "timeline-bottom-box": string;
    "timeline-box": string;
    "timeline-box-labels": string;
    "timeline-label": string;
    "timeline-line": string;
    "timeline-pos": string;
    "timeline-rank": string;
    "timeline-title": string;
    "timeline-warning": string;
    timelineBottomBox: string;
    timelineBox: string;
    timelineBoxLabels: string;
    timelineLabel: string;
    timelineLine: string;
    timelinePos: string;
    timelineRank: string;
    timelineTitle: string;
    timelineWarning: string;
    topConstructedRank: string;
    topLimitedRank: string;
    topLogoAbout: string;
    topNav: string;
    topNavIcon: string;
    topNavIcons: string;
    topNavInfo: string;
    topNavItem: string;
    topNavItemNoLabel: string;
    topNavItemText: string;
    topPatreon: string;
    topStatus: string;
    topStatusPop: string;
    topStatusSpan: string;
    topSync: string;
    topUserdataContainer: string;
    topUsername: string;
    topUsernameId: string;
    topWildcardsCont: string;
    topWildcardsSetIcon: string;
    topWildcardsSetsCont: string;
    topWildcardsWcIcon: string;
    top_constructed_rank: string;
    top_limited_rank: string;
    top_logo_about: string;
    top_nav: string;
    top_nav_icon: string;
    top_nav_icons: string;
    top_nav_info: string;
    top_nav_item: string;
    top_nav_item_no_label: string;
    top_nav_item_text: string;
    top_patreon: string;
    top_status: string;
    top_status_pop: string;
    top_status_span: string;
    top_sync: string;
    top_userdata_container: string;
    top_username: string;
    top_username_id: string;
    top_wildcards_cont: string;
    top_wildcards_set_icon: string;
    top_wildcards_sets_cont: string;
    top_wildcards_wc_icon: string;
    touCell: string;
    touCenter: string;
    touCheck: string;
    touCheckCont: string;
    touChecks: string;
    touContA: string;
    touContB: string;
    touContC: string;
    touContainer: string;
    touCreateBut: string;
    touDesc: string;
    touFlag: string;
    touListPlayerName: string;
    touListPlayerNameTitle: string;
    touMatchCont: string;
    touMatchP: string;
    touMatchScore: string;
    touName: string;
    touOpp: string;
    touOppSub: string;
    touRecord: string;
    touReload: string;
    touRoundCont: string;
    touRoundTitle: string;
    touScoreWin: string;
    touStandCell: string;
    touStandLine: string;
    touStandLineTitle: string;
    touStandName: string;
    touStandSmall: string;
    touState: string;
    touTab: string;
    touTabSelected: string;
    touTabsCont: string;
    touTop: string;
    touTopCont: string;
    touTopStatus: string;
    tou_cell: string;
    tou_center: string;
    tou_check: string;
    tou_check_cont: string;
    tou_checks: string;
    tou_cont_a: string;
    tou_cont_b: string;
    tou_cont_c: string;
    tou_container: string;
    tou_create_but: string;
    tou_desc: string;
    tou_flag: string;
    tou_list_player_name: string;
    tou_list_player_name_title: string;
    tou_match_cont: string;
    tou_match_p: string;
    tou_match_score: string;
    tou_name: string;
    tou_opp: string;
    tou_opp_sub: string;
    tou_record: string;
    tou_reload: string;
    tou_round_cont: string;
    tou_round_title: string;
    tou_score_win: string;
    tou_stand_cell: string;
    tou_stand_line: string;
    tou_stand_line_title: string;
    tou_stand_name: string;
    tou_stand_small: string;
    tou_state: string;
    tou_tab: string;
    tou_tab_selected: string;
    tou_tabs_cont: string;
    tou_top: string;
    tou_top_cont: string;
    tou_top_status: string;
    tourneyPwdContainer: string;
    tourney_pwd_container: string;
    troubleLink: string;
    trouble_link: string;
    twitterLink: string;
    twitter_link: string;
    typeArt: string;
    typeCre: string;
    typeEnc: string;
    typeIcon: string;
    typeIconCont: string;
    typeIns: string;
    typeLan: string;
    typePla: string;
    typeSor: string;
    type_art: string;
    type_cre: string;
    type_enc: string;
    type_icon: string;
    type_icon_cont: string;
    type_ins: string;
    type_lan: string;
    type_pla: string;
    type_sor: string;
    typesContainer: string;
    types_container: string;
    uncommon: string;
    unlink: string;
    uxItem: string;
    ux_item: string;
    versionNumber: string;
    version_number: string;
    visualView: string;
    wanted: string;
    wcBooster: string;
    wcCommon: string;
    wcComplete: string;
    wcCost: string;
    wcDeckstabCost: string;
    wcExploreCost: string;
    wcMythic: string;
    wcRare: string;
    wcSearchIcon: string;
    wcUncommon: string;
    wc_booster: string;
    wc_common: string;
    wc_complete: string;
    wc_cost: string;
    wc_deckstab_cost: string;
    wc_explore_cost: string;
    wc_mythic: string;
    wc_rare: string;
    wc_search_icon: string;
    wc_uncommon: string;
    wildcardsCost: string;
    wildcards_cost: string;
    wnewImage: string;
    wnewImageCont: string;
    wnewImageDesc: string;
    wnewImageTitle: string;
    wnewNext: string;
    wnewPosBall: string;
    wnewPosBallSelected: string;
    wnewPrev: string;
    wnewScroller: string;
    wnewScrollerPosCont: string;
    wnewSubVersion: string;
    wnewTitle: string;
    wnew_image: string;
    wnew_image_cont: string;
    wnew_image_desc: string;
    wnew_image_title: string;
    wnew_next: string;
    wnew_pos_ball: string;
    wnew_pos_ball_selected: string;
    wnew_prev: string;
    wnew_scroller: string;
    wnew_scroller_pos_cont: string;
    wnew_sub_version: string;
    wnew_title: string;
    wrapperColumn: string;
    wrapper_column: string;
  }
}

declare const IndexCssModule: IndexCssModule.IIndexCss & {
  /** WARNING: Only available when `css-loader` is used without `style-loader` or `mini-css-extract-plugin` */
  locals: IndexCssModule.IIndexCss;
};

export = IndexCssModule;
