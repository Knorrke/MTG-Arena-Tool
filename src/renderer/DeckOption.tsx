/*
  Not used / deprecated?
*/
import React from "react";
import { getDeckName, deckExists } from "../shared/store";
import indexCss from "./index.css";
import sharedCss from "../shared/shared.css";
import { WHITE, BLUE, BLACK, RED, GREEN, COLORLESS } from "../shared/constants";

const manaClasses: string[] = [];
manaClasses[WHITE] = sharedCss.manaW;
manaClasses[BLUE] = sharedCss.manaU;
manaClasses[BLACK] = sharedCss.manaB;
manaClasses[RED] = sharedCss.manaR;
manaClasses[GREEN] = sharedCss.manaG;
manaClasses[COLORLESS] = sharedCss.manaC;

export interface DeckOptionDeck {
  colors?: number[];
  name?: string;
  archived?: boolean;
}

export interface DeckOptionProps {
  deckId: string;
  deck: DeckOptionDeck;
}

export default function DeckOption(props: DeckOptionProps): JSX.Element {
  const { deckId, deck } = props;

  const exists = deckExists(deckId);
  const deckName: string = exists ? getDeckName(deckId) : deck.name || "";
  let maxChars = 10;
  if (exists && deck.colors) {
    maxChars = 16 - 2 * deck.colors.length;
  }

  return (
    <>
      {deckName.length > maxChars ? (
        <abbr title={deckName}>{deckName.slice(0, maxChars)}...</abbr>
      ) : (
        deckName
      )}
      {exists ? (
        <>
          {deck.archived && (
            <small>
              <i>(archived)</i>
            </small>
          )}
          <div className={indexCss.flexItem}>
            {deck.colors &&
              deck.colors.map((color) => (
                <div
                  className={sharedCss.manaS16 + " " + manaClasses[color - 1]}
                  key={color}
                />
              ))}
          </div>
        </>
      ) : (
        <small>
          <i>(deleted)</i>
        </small>
      )}
    </>
  );
}
