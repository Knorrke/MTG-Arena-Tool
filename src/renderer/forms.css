.form-input-container {
  display: flex;
  height: 32px;
  width: 100%;
  padding: 4px 0 12px 0;
}

.form-input-container input {
  background-color: var(--color-mid-40);
  border: 1px solid var(--color-mid-50);
  display: inline-table;
  border-radius: 3px;
  color: var(--color-back);
  height: 22px;
  width: 100%;
  font-size: 15px;
  padding: 6px;
  user-select: none;
  font-family: var(--main-font-name);
}

.form-input-container input:focus {
  border: 1px solid white;
  outline-style: none;
}

.form-label {
  font-family: var(--main-font-name);
  color: var(--color-back);
  margin: auto 0;
}

.form-options {
  width: 18.9em;
  margin: 0 auto auto auto;
  z-index: 5;
  display: flex;
  flex-direction: column;
}

.form-container {
  margin: auto;
  flex-grow: 2;
  display: flex;
  width: -webkit-fill-available;
  height: -webkit-fill-available;
  flex-direction: column;
}

.form-icon {
  background-image: url(../assets/icons/icon-256.png);
  background-color: var(--color-light);
  border-radius: 7em;
  background-size: contain;
  width: 7em;
  height: 7em;
  position: relative;
  margin: auto auto -7em auto;
  top: -7em;
}

.form-authenticate {
  max-width: 18.9em;
  z-index: 5;
  width: -webkit-fill-available;
  margin: auto auto 16px auto;
  background-color: var(--color-light);
  padding: 3.5em 2em 2em 2em;
  border-radius: 3px;
  box-shadow: 0px 2px 7px 4px rgba(0, 0, 0, 0.3);
}

.form-button {
  align-self: center;
  cursor: pointer;
  width: -webkit-fill-available;
  width: -moz-available;
  height: 36px;
  border-radius: 3px;
  color: var(--color-light);
  font-size: 16px;
  text-align: center;
  border: 1px solid var(--color-mid-50);
  background-color: var(--color-back);
  -webkit-transition: all .1s ease-in;
  font-family: var(--main-font-name);
  display: block;
  margin: auto;
}

.form-button:disabled {
  background-color: var(--color-mid-50);
  color: var(--color-light-50);
}

.form-button:hover:enabled {
  background-color: var(--color-mid-66);
  box-shadow: 0px 6px 15px 0px rgba(0, 0, 0, 0.1);
  border: 1px solid var(--color-mid-50);
}

.form-button:active:enabled {
  background-color: var(--color-mid);
  line-height: 34px;
  box-shadow: 0px 3px 8px 0px rgba(0, 0, 0, 0.6);
}

.form-error {
  text-align: center;
  line-height: 32px;
  font-family: var(--main-font-name-bold-it);
  color: var(--color-r);
  width: 19em;
  height: 32px;
  margin: 0px auto;
  -webkit-transition: all .3s ease-in;
}