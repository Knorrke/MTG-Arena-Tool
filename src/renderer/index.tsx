import "../shared/shared.css";
import "./index.css";
import "@github/time-elements";
import RenderApp from "./app/renderApp";

RenderApp();
