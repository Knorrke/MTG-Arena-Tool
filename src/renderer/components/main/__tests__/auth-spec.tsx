/* eslint-env jest */
import renderer from "react-test-renderer";

import React from "react";
import Auth from "../Auth";
import { Provider } from "react-redux";
import store from "../../../../shared/redux/stores/rendererStore";

describe("Auth component", () => {
  const form = { email: "jester@gmail.com", pass: "fake", rememberme: false };

  it("renders properly", () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Auth authForm={form} />
        </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
