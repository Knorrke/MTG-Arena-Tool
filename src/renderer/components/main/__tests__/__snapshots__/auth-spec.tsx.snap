// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auth component renders properly 1`] = `
<div
  className="form-container"
>
  <div
    className="form-authenticate"
  >
    <div
      className="form-icon"
    />
    <div
      id="loginform"
    >
      <label
        className="form-label"
      >
        Email
      </label>
      <div
        className="form-input-container"
      >
        <input
          autoComplete="off"
          id="signin_email"
          onChange={[Function]}
          type="email"
          value="jester@gmail.com"
        />
      </div>
      <label
        className="form-label"
      >
        Password
      </label>
      <div
        className="form-input-container"
      >
        <input
          autoComplete="off"
          id="signin_pass"
          onChange={[Function]}
          type="password"
          value="fake"
        />
      </div>
      <div
        style={
          Object {
            "color": "var(--color-mid-75)",
            "cursor": "pointer",
            "marginBottom": "16px",
          }
        }
      >
        <a
          className="forgot_link"
          onClick={[Function]}
        >
          Forgot your password?
        </a>
      </div>
      <button
        className="form-button"
        disabled={false}
        id="submit"
        onClick={[Function]}
        type="submit"
      >
        Login
      </button>
      <div
        className="form-error"
      >
        
      </div>
    </div>
  </div>
  <div
    className="form-options"
  >
    <label
      className="check_container hover_label"
      onClick={[Function]}
      style={
        Object {
          "margin": "auto auto 12px auto",
          "width": "max-content",
        }
      }
    >
      Remember me?
      <input
        checked={false}
        disabled={true}
        type="checkbox"
      />
      <span
        className="checkmark"
      />
    </label>
    <div
      className="message_small"
    >
      Dont have an account?
       
      <a
        className="signup_link"
        onClick={[Function]}
      >
        Sign up!
      </a>
    </div>
    <div
      className="message_small"
    >
      You can also
       
      <a
        className="offline_link"
        onClick={[Function]}
      >
        continue offline
      </a>
    </div>
  </div>
</div>
`;
